server/paymentProviders/stripe/common.ts

```diff
@@ -174,7 +174,7 @@ export const createChargeTransactions = async (
     platformFeeInHostCurrency = fees.applicationFee;
   }
 
-  const clearedAt = charge.created ? moment.unix(charge.created).toDate() : null;
+  const clearedAt = balanceTransaction.available_on ? moment.unix(balanceTransaction.available_on).toDate() : null;
```

Comment:
Are we confident `balanceTransaction.available_on` is always available? Considering the type version we use locally and the version of the API we're on, I suggest keep the `charge.created` as fallback before getting to null.